dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -1760,13 +1796,14 @@ private void setPoolIncrementDecrement(int poolSize) {
          * decrement exactly to coreThreads.
          */
 
```

Comment:
Should the comments and trace be updated here to reference currentMinimumPoolSize instead of coreThreads?

Reply:
Done